src/python/bot/minimizer/minimizer.py

```diff
@@ -208,18 +208,21 @@ def _delete_file_if_needed(self, input_file):
     except OSError:
       pass
 
-  def _report_progress(self):
+  def _report_progress(self, is_final=False):
```

Comment:
Nit: is_final isn't too clear a name for what this actually does. Maybe something like force_progress_report?